main/webapp/modules/core/scripts/views/data-table/cell-ui.js

```diff
@@ -96,16 +96,39 @@ DataTableCellUI.prototype._render = function() {
       nonstringSpan.className = 'data-table-value-nonstring';
       nonstringSpan.textContent = cell.v;
       divContent.appendChild(nonstringSpan);
-    } else if (URL.looksLikeUrl(cell.v)) {
-      var url = document.createElement('a');
-      url.textContent = cell.v;
-      url.setAttribute('href', cell.v);
-      url.setAttribute('target', '_blank');
-      divContent.appendChild(url);
-    } else {
-      var span = document.createElement('span');
-      span.textContent = cell.v;
-      divContent.appendChild(span);
+    } 
+    else{
+      var arr = cell.v.split(" ");
+      var spanArr =[];
+      for( var i=0;i<arr.length;i++){
```

Comment:
```suggestion
      for (var i=0; i<arr.length; i++) {
```
Until we adopt a new coding style for our Javascript, let's match the current style for code added to an existing module